docker-compose.yaml

```diff
@@ -4,23 +4,20 @@ services:
   api:
     build: mystbin/backend
     ports:
-      - 127.0.0.1:5557:9000/tcp
```

Comment:
By default this runs on `0.0.0.0`, hence the explicit loopback address assignment.